bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obscomp/SPInstObsComp.java

```diff
@@ -435,24 +435,25 @@ public Instrument getInstrument() {
     // REL-1678: 7 seconds DHS write overhead
     // REL-1934: 10 seconds DHS write overhead for F2
 
-    public static final Map<Instrument, PlannedTime.CategorizedTime> DHS_WRITE_TIMES;
+    public static final Map<Instrument, PlannedTime.CategorizedTime> WRITE_TIMES;
```

Comment:
Will this affect serial version? @swalker2m

Reply:
Yeah I think this breaks serialization.  Do we need to deploy it before the June release in the production ODB?  If so we have to think about this and probably make some changes.  Otherwise, it's fine since we break serialization with every major release anyway and rely on XML migration.

Reply:
Hmm, well I guess this is `static` so maybe not an issue.  We'll need to test carefully though in general if this is to go into production before June.